cumulusci/tasks/github/publish.py

```diff
@@ -221,15 +213,26 @@ def _create_release(self, path, commit):
             tagger={
                 "name": self.github_config.username,
                 "email": self.github_config.email,
-                "date": "{}Z".format(datetime.utcnow().isoformat()),
+                "date": f"{datetime.utcnow().isoformat()}Z",
             },
             lightweight=False,
         )
 
         # Create the release
         self.target_repo.create_release(
             tag_name=self.tag_name,
-            name=self.options["version"],
+            name=self._get_release_name_from_tag(self.tag_name),
             prerelease=release.prerelease,
             body=release_body,
         )
+
+    def _get_release_name_from_tag(self, tag_name):
```

Comment:
Github ate the other substantive question from the above review: Why wouldn't `self.project_config.get_version_for_tag(tag_name)` work here?